docs/api/structures/usb-device.md

```diff
@@ -0,0 +1,8 @@
+# USBDevice Object
+
+* `deviceId` string - Unique identifier for the device.
+* `name` string - Name of the device.
+* `vendorId` Integer - The USB vendor ID.
+* `productId` Integer - The USB product ID.
+* `serialNumber` string (optional) - The USB device serial number.
+* `ephemeralGuid` string (optional)  - Temporary unique identifier devices that do not have a serial number.
```

Comment:
There are no other docs or fiddles on how to use `ephemeralGuid`. Maybe this requires more explanation -- e.g. would it be possible to add a Fiddle showing how an app dev would use this property in practice?

Reply:
It was a carry over from the Chromium implementation that we don't really need, so I removed it.